packages/evm/src/types.ts

```diff
@@ -343,9 +343,13 @@ export interface EVMOpts {
   stateManager?: StateManagerInterface
 
   /**
+   * The EVM comes with a basic mock blockchain interface and implementation for
+   * non-block containing use cases.
    *
+   * For block-containing setups use the full blockchain implementation from the
```

Comment:
The only use case in the EVM for Blockchain is to use the BLOCKHASH opcode. So, in a context where BLOCKHASH is not being used / ran, using the mockchain is fine

Reply:
Yeah, I would nevertheless not want to write something like "if you use BLOCKHASH" here since in many cases this is just not known before.

So I would think this phrasing is fine, people will know if they have/need a blockchain containing real data at hand.